src/graphql/models/Cooccurrence.js

```diff
@@ -0,0 +1,41 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLID,
+} from 'graphql';
+import Article from './Article';
+import Node from '../interfaces/Node';
+
+const Cooccurrence = new GraphQLObjectType({
+  name: 'Cooccurrence',
+  interfaces: [Node],
+  fields: () => ({
+    id: { type: new GraphQLNonNull(GraphQLID) },
+    userId: { type: GraphQLNonNull(GraphQLString) },
+    appId: { type: GraphQLNonNull(GraphQLString) },
+    articles: {
+      type: new GraphQLList(Article),
+      resolve: async ({ articleIds }, args, { loaders }) =>
+        loaders.searchResultLoader.load({
+          index: 'articles',
+          type: 'doc',
+          body: {
+            query: {
+              ids: {
+                values: articleIds,
+              },
+            },
+          },
+        }),
+    },
+    articleIds: {
+      type: new GraphQLList(new GraphQLNonNull(GraphQLString)),
```

Comment:
Since `cooccurrences` always come with multiple article ids; null `articleIds` should probably considered as an error. We can add `GraphQLNonNull` to the list too.

We can use apply `GraphQLNonNull` to `articles` fields too -- article ID not resolving to an article is probably an error that should be blocked on API side.